content/authorization/tls-guidance.md

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
Shall we have a section that calls out what not to do / bad practices? I'm thinking we should list out that developers should not store a copy of our cert.

Reply:
I'm open to other's opinions, here. 

My personal opinion is that it is unnecessary.  If there were more instances of this occurring, it would certainly be justified.  Most developers are on the correct path (using an evergreen TLS implementation provided by their platform, as we recommend in this guidance.)  Adding a mountain of "don't do's" could just make developers more apprehensive.